web/src/features/map-controls/ZoomControls.tsx

```diff
@@ -1,15 +1,30 @@
-import { useFeatureFlag } from 'features/feature-flags/api';
-import { ReactElement } from 'react';
+
+import { ReactElement, useEffect, useState } from 'react';
 import { NavigationControl } from 'react-map-gl/maplibre';
 
 export default function ZoomControls(): ReactElement {
-  const isConsumptionOnlyMode = useFeatureFlag('consumption-only');
-  const marginTop = isConsumptionOnlyMode ? 54 : 98;
+  const [topMargin, setTopMargin] = useState(98);
+  useEffect(() => {
+    const updateTopMargin = () => {
+      const safeAreaTop =
+        Number.parseInt(
+          getComputedStyle(document.documentElement).getPropertyValue('--sat'),
+          10
+        ) || 0;
+      setTopMargin(98 + safeAreaTop);
+    };
+
+    updateTopMargin();
+    window.addEventListener('resize', updateTopMargin);
+    return () => window.removeEventListener('resize', updateTopMargin);
+  }, []);
```

Comment:
This should use a the resizeObserverHook, useEffects should be options of last resort in react. Using a event listener instead of a resizeObserver also have a performance overhead.